plugins/toolkit/jetbrains-ultimate/build.gradle.kts

```diff
@@ -29,3 +29,7 @@ dependencies {
 intellijToolkit {
     ideFlavor.set(IdeFlavor.IU)
 }
+
+intellij {
```

Comment:
Would this work if we added it to the top-level build file?

Reply:
no